.github/workflows/conformance-eks.yaml

```diff
@@ -76,10 +74,8 @@ jobs:
     name: Deduce required tests from code changes
     if: |
       (github.event_name == 'issue_comment' && (
-        startsWith(github.event.comment.body, 'ci-eks') && !startsWith(github.event.comment.body, 'ci-eks-') ||
-        startsWith(github.event.comment.body, '/ci-eks') && !startsWith(github.event.comment.body, '/ci-eks-') ||
-        startsWith(github.event.comment.body, 'test-me-please') && !startsWith(github.event.comment.body, 'test-me-please-') ||
-        startsWith(github.event.comment.body, '/test') && !startsWith(github.event.comment.body, '/test-')
```

Comment:
The commit message mentions that we are dropping two types of support, I guess dropping `test-` falls into the second type?

Reply:
`!startsWith(github.event.comment.body, '/test-')` was there to ensure we don't trigger this workflow when trying to trigger the backport workflows (e.g., via `/test-backport-1.11`). Since we're now checking the comment is exactly equal to `/test`, we don't need this anymore.